core/controllers/reader_test.py

```diff
@@ -1365,6 +1366,26 @@ def test_exp_incomplete_event_handler(self):
             learner_progress_services.get_all_partially_learnt_topic_ids(
                 self.user_id), [self.TOPIC_ID])
 
+        # If the exploration is played in context of an invalid story, raise
+        # an error.
+
+        def _mock_none_function(_):
+            """Mocks None."""
+            return None
+
+        story_fetchers_swap = self.swap(
+            story_fetchers, 'get_story_by_id', _mock_none_function)
+
+        with story_fetchers_swap:
+            with self.capture_logging(min_level=logging.ERROR) as captured_logs:
+                self.post_json(
+                    '/explorehandler/exploration_maybe_leave_event/%s' % self.EXP_ID_2_0, # pylint: disable=line-too-long
```

Comment:
Don't use pragma. Can't the Url be split into multiple lines, since it is just a string?

Reply:
Done